themes/SuiteR/css/style.css

```diff
@@ -6067,17 +6066,101 @@ ul.clickMenu.searchAppliedAlert {
     padding: 0;
 }
 
+
+@media (max-width: 992px) {
+    ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li.desktopOnly {
+        display: none;
+    }
+
+    .desktopOnly {
+        display: none;
+    }
+}
+
+@media (min-width: 992px) {
+    ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li.mobileOnly {
+        display: none;
+    }
+
+    .mobileOnly {
+        display: none;
+    }
+}
+
+
+#selectedRecordsTop,
+#selectedRecordsBottom {
+    background-position-x: 95%;
+    padding-right: 10px;
+    color: #444444;
+    display: block!important;
+}
+
+ul.clickMenu.searchAppliedAlert.selectmenu.SugarActionMenu.listViewLinkButton {
+    padding: 0;
+}
+
+ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li.desktopOnly a {
+    line-height: 15px;
+}
+
+ul.listViewLinkButton > li {
+    padding: 0;
+}
+
+ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li {
+    padding: 0;
+}
+
+ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li > a.glyphicon-remove {
+    font-weight: bold;
+}
+
+ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li.desktopOnly > a.glyphicon-remove {
+    -webkit-border-radius: 2px;
+    -moz-border-radius: 2px;
+    border-radius: 2px;
+    line-height: 0.8;
+    font-weight: bold;
+}
+
+ul.clickMenu.selectmenu.SugarActionMenu.listViewLinkButton > li.desktopOnly > a.glyphicon-remove:before {
+    content: '';
+}
+
 .paginationActionButtons .parent-dropdown-handler {
-    padding: 3px 10px;
+    padding: 0 0 0 5px;
 }
 
-.paginationActionButtons .parent-dropdown-handler:hover * {
-    color: #3C8DBC;
+.paginationActionButtons .parent-dropdown-handler label {
+    margin-bottom: 0px;
+    height: 25px;
+    line-height: 25px;
+    color: #444444;
+}
+
+#selectLinkTop .ddopen {
+    top: 25px!important;
```

Comment:
The issue:

![screenshot_20170127_094523](https://cloud.githubusercontent.com/assets/12231216/22366642/b54adf4c-e475-11e6-9016-578413971bb2.png)

The code:
`<ul id="" class="subnav ddopen" style="display: block; top: 35px;">`

As you can see the top is being set in an inline style by include/javascript/jquery/jquery.sugarMenu.js: 218 / 221

The solution:

The only way to override an inline style is to use !important.

![screenshot_20170127_094617](https://cloud.githubusercontent.com/assets/12231216/22366680/df777258-e475-11e6-90af-72aa8aecfa17.png)